static/js/browser2/statvar_group_node.tsx

```diff
@@ -21,22 +21,31 @@
 import React from "react";
 import _ from "lodash";
 import Collapsible from "react-collapsible";
-import { StatVarGroupNodeType, StatVarHierarchyNodeType } from "./util";
+import {
+  StatVarGroupNodeType,
+  StatVarHierarchyNodeType,
+  StatVarNodeType,
+} from "./util";
 import { StatVarNode } from "./statvar_node";
 
+const SCROLL_DELAY = 400;
+const BULLET_POINT_HTML = <span>&#8226;</span>;
+const DOWN_ARROW_HTML = <span>&#x25BC;</span>;
+const RIGHT_ARROW_HTML = <span>&#x25BA;</span>;
+
 interface StatVarGroupNodePropType {
-  dcid: string;
+  placeDcid: string;
   placeName: string;
   statVarGroupId: string;
   data: { [key: string]: StatVarGroupNodeType };
   pathToSelection: string[];
   specializedEntity?: string;
   isSelected: boolean;
-  isOpened: boolean;
+  open: boolean;
```

Comment:
please add short comments for these props, so that it's clear to callers what's required

Reply:
done